templates/accounts/registration.html

```diff
@@ -3,10 +3,22 @@
 {% block title %}registration{% endblock %}
 
 {% block section_content %}
+<style> #username_check { display:none; color: #bd2d33; font-style: italic; }</style>
 <script type="text/javascript">
     $(function () {
-    	$("#id_username").focus()
-    });
+      $("#id_username").focus()
+      
+      $('#id_username').parent().append('<div id="username_check">The username is not available</div>');
```

Comment:
Can you please change this to "Sorry, this username is already being used by someone else."?